srcpkgs/lhasa/template

```diff
@@ -0,0 +1,40 @@
+# Template file for 'lhasa'
+pkgname=lhasa
+version=0.3.1
+revision=1
+build_style=gnu-configure
+hostmakedepends="automake libtool"
+depends="liblhasa>=${version}_${revision}"
+checkdepends="tzdata"
+short_desc="Free Software LHA implementation"
+maintainer="Hill Ma <maahiuzeon@gmail.com>"
+license="ISC"
+homepage="https://github.com/fragglet/lhasa"
+distfiles="https://github.com/fragglet/lhasa/releases/download/v${version}/lhasa-${version}.tar.gz"
+checksum=3fb08e5f85a9b9dd023922896be9157d5fb5c0448424681810aaa2b0558a5f24
+
+if [ "$CROSS_BUILD" ]; then
+	configure_args+=" --host=${XBPS_CROSS_TRIPLET}"
+fi
```

Comment:
Are you sure this is necessary? We do it in the build style already...

Reply:
Removed in 3cfe609. Not sure why all the native builds fail now :(